services/PA-accounts-service/src/config.ts

```diff
@@ -13,7 +13,7 @@ export const URL_121_SERVICE = process.env.URL_121_SERVICE_API;
 export const URL_USERIMS = 'http://11.0.0.5:50003/api';
 
 let _walletPasswordEncryptionKey: string;
-if (process.env.NODE_ENV == 'production' || process.env.NODE_ENV == 'staging') {
+if (['production', 'staging', 'test'].indexOf(process.env.NODE_ENV) > -1) {
```

Comment:
I think we should remove this decision/dependency-on-NODE_ENV entirely.
We could base it on the value/existence of the `WALLET_PASSWORD_ENCRYPTION_KEY`-variable.
(Or add another like `ENABLE_ENCRYPTION`, but that doesn't make a lot of sense).